api/src/main/kotlin/com/google/prefab/api/Module.kt

```diff
@@ -105,7 +105,7 @@ class Module(val path: Path, val pkg: Package) {
         path.resolve("libs").toFile().listFiles()?.map { directory ->
             val basename = directory.toPath().fileName.toString()
             val components = basename.split(".", limit = 2)
-            if (components.size != 2) {
+            if (components.size != 2 || components.first() == "") {
```

Comment:
Instead of this, wdyt of:

```kotlin
if (components.first().isEmpty()) {
    throw RuntimeException("Found unexpected file in libs directory: $basename")
}
```

Strictly speaking `.DS_store` isn't missing its artifact ID, it's missing its platform ID. Anything without a platform ID is invalid, so probably better to be more clear about that.

Reply:
I've added a couple of exceptions and tests to handle the cases where: 

1) the directory name is invalid (it does not contain a string which can be split with a ".")
2) the platform ID is missing